bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/BoschHttpClient.java

```diff
@@ -194,7 +243,12 @@ public Request createRequest(String url, HttpMethod method) {
      * @return created HTTP request instance
      */
     public Request createRequest(String url, HttpMethod method, @Nullable Object content) {
-        Request request = this.newRequest(url).method(method).header("Content-Type", "application/json");
+        logger.trace("Create request for http client {}", this.toString());
+
+        Request request = this.newRequest(url).method(method).header("Content-Type", "application/json")
+                .header("api-version", "2.1") // see https://github.com/BoschSmartHome/bosch-shc-api-docs/issues/46
```

Comment:
Would this cause any problems with devices using older apis?

Reply:
Not as far as I know @GerdZanker Did you read any information about this?

Reply:
Thank you @cpmeister for sharing your experience regarding API compatibility.

The update to 2.1 was rolled out in Dezember 2020 and is installed automatically within weeks. I assume that we not run into issues after more than two months. 

Statement from the BoschSmartHome maintainers is:

>Only very, very few might not have API 2.1 available by now

The actual supported api version is also logged during binding initialization and therefore we can detect old Smart Home Controllers and suggest to do an update.